### PR TITLE
ci: run ruff check

### PR DESCRIPTION
**Pull request by nicoloboschi**

### CODE REVIEW

File: libs/ragulate/ragstack_ragulate/cli.py [hunk 1]
```diff
@@ -1,5 +1,4 @@
 # this must be imported first for dynamic module loading to function properly
-import setuptools  # isort: skip
```
Comment (epinzur) [inline libs/ragulate/ragstack_ragulate/cli.py:2]: we need to keep this one

Comment (epinzur) [reply libs/ragulate/ragstack_ragulate/cli.py:2]: @nicoloboschi there is prob a way for ruff to ignore lines

Comment (nicoloboschi) [reply libs/ragulate/ragstack_ragulate/cli.py:2]: added back

**Review by cbornet — APPROVED** [review]

File: tox.ini [hunk 2]
```diff
@@ -47,18 +47,19 @@ commands =
 description = lint
 skip_install = true
 commands =
-    poetry install --no-root --only lint
-    poetry run ruff format --diff ragstack tests
-    poetry run ruff check --select I ragstack tests
-    poetry run ruff check ragstack tests
-    poetry run yamllint -s -c .github/.yamllint .github/
+    ruff format --diff .
```
Comment (cbornet) [inline tox.ini:50]: Please keep the changes from main

**Review by cbornet — CHANGES_REQUESTED** [review]

**Review by cbornet — APPROVED** [review]

**Pull request merged**